providers/snowflake/src/airflow/providers/snowflake/hooks/snowflake.py

```diff
@@ -253,7 +253,7 @@ def _get_conn_params(self) -> dict[str, str | None]:
 
         This is used in ``get_uri()`` and ``get_connection()``.
         """
-        conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined]
+        conn = self.get_connection(self.snowflake_conn_id)
```

Comment:
@gopidesupavan +1 for https://github.com/apache/airflow/pull/53252#discussion_r2202912638